docsite/rst/intro_windows.rst

```diff
@@ -242,6 +289,100 @@ Looking at an Ansible checkout, copy the `examples/scripts/upgrade_to_ps3.ps1 <h
 
 .. _what_windows_modules_are_available:
 
+Verifying Windows Configuration
+```````````````````````````````
+
+After you have set up the Windows host you can use the following commands to verify the setup to ensure Ansible can communite with the host.
+
+Verify WinRM Listeners
+++++++++++++++++++++++
+
+To see what listeners are currently active you can run ``winrm enumerate winrm/config/listener`` the following command in PowerShell::
```

Comment:
I think:

> 
you can run ``winrm enumerate winrm/config/listener`` the following command in PowerShell::

should be:

> 
you can run the following ```winrm``` command in PowerShell::